### PR TITLE
AttachSharedSDToLocalDC: Add feature page.

### PR DESCRIPTION
**Pull request by maorlipchuk**
Changes proposed in this pull request:

- Introducing a new feature page for shared storage domains in local Data Center, describing all the new functionalities and changes

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)
@mlipchuk

This pull request needs review by: (please @mention the reviewer if relevant)
@rollandf @mureinik

### CODE REVIEW

**Comment by maorlipchuk** [comment]
@rollandf @mureinik

**Comment by rollandf** [comment]
@maorlipchuk you should fill up the first comment as required, short description and signing by your github avatar

File: source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Owner
+
+- Name: [Maor Lipchuk](https://github.com/maorlipchuk)
+- Email: <mlipchuk@redhat.com>
+
+## Detailed Description
+
+As part of this feature the user will now have the ability to change an initialized Data Center type (Local vs Shared).
+The following updates will be avialiable:
```
Comment (rollandf) [inline source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md]: typo avialiable->available

File: source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+1. Shared to Local - Only for a Data Center that does not contain more than one Host and more than one cluster, since local Data Center does not support it.
+The engine should validate and block this operation with the following messages:
+* CLUSTER_CANNOT_ADD_MORE_THEN_ONE_HOST_TO_LOCAL_STORAGE
+* VDS_CANNOT_ADD_MORE_THEN_ONE_HOST_TO_LOCAL_STORAGE
+2. Local to Shared - Only for a Data Center that does not contain a local Storage Domain.
+The engine should validate and block this operation with the following message ERROR_CANNOT_CHANGE_STORAGE_POOL_TYPE_WITH_LOCAL.
+
+### Functionality
+
+#### Detach shared Stoage Domain from local Data Center
```
Comment (rollandf) [inline source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md]: typo Stoage->Storage

File: source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Shared SDs behave differently, since those can be detached.
+Based on that, if a local DC will be removed the master Storage Domain will be formatted only if it is a local SD (to keep the previous behaviour) and if it is a shared SD then this Storage Domain will only be detached.
+
+#### Memory volume placement
+When selecting a Storage Domain to place memory volumes, there are currently three comparators that choose the target Storage Domain:
+1. The number of the VM's disks in the Storage Domain - prefer the Storage Domain where the VM has the most disks
+2. Storage type - prefer file storage over block storage
+3. Free space - prefer Storage Domains with more free space 
+
+#### Reconstruct
```
Comment (rollandf) [inline source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md]: This title seems to be misplaced

**Comment by mureinik** [comment]
@maorlipchuk ping?

File: source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+Shared SDs behave differently, since those can be detached.
+Based on that, if a local DC will be removed the master Storage Domain will be formatted only if it is a local SD (to keep the previous behaviour) and if it is a shared SD then this Storage Domain will only be detached.
+
+#### Memory volume placement
+When selecting a Storage Domain to place memory volumes, there are currently three comparators that choose the target Storage Domain:
+1. The number of the VM's disks in the Storage Domain - prefer the Storage Domain where the VM has the most disks
+2. Storage type - prefer file storage over block storage
+3. Free space - prefer Storage Domains with more free space 
+
+#### Re-Initialize Data Center
```
Comment (rollandf) [inline source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md]: This title shouldn't be on line 50 ?

Comment (maorlipchuk) [inline source/develop/release-management/features/storage/sharedStorageDomainsAttachedToLocalDC.html.md]: yes

**Comment by rollandf** [comment]
👍

**Pull request merged**